forge-ai/src/main/java/forge/ai/simulation/GameSimulator.java

```diff
@@ -33,7 +33,8 @@ public class GameSimulator {
     public GameSimulator(SimulationController controller, Game origGame, Player origAiPlayer, PhaseType advanceToPhase) {
         this.controller = controller;
         copier = new GameCopier(origGame);
-        simGame = copier.makeCopy(advanceToPhase, origAiPlayer);
+        simGame = copier.makeCopy();
+        // simGame.advanceToPhase(advanceToPhase, origAiPlayer)
```

Comment:
Don't understand this change
it's important for simulation AI to look at future combat

Reply:
Oh i commented this out for other reasons, but should be able to add it back in. I don't think we had any tests that covered this path so it was hard to see where we used it. 

I don't think its related to the disabled test though. That seems to be a problem with the simulation assuming cards would be ordered in the same way from the original game to the sim games. i.e. Why I had to change all of the other loops from index to findById(). The AI seems to be trying to target the opposing creature in that circumstance and I couldn't figure out where exactly that wrong decision tree was occurring.